src/llmq/utils.cpp

```diff
@@ -26,7 +26,6 @@
 #include <atomic>
 #include <optional>
 
-// TODO remove this const
```

Comment:
8454b58c76339269edfc90ceedc3990e2181a71f I believe we added this special logic to avoid needing to re-hardfork testnet or something like that. I forget